sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeHealthcareEntitiesTests.cs

```diff
@@ -0,0 +1,230 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace Azure.AI.TextAnalytics.Tests
+{
+    public class RecognizeHealthcareEntitiesTests : TextAnalyticsClientLiveTestBase
+    {
+        public RecognizeHealthcareEntitiesTests(bool isAsync) : base(isAsync) { }
```

Comment:
tests that seem to be missing:
- checking link entities and its values,
- values for Entity are populated
- relations are correctly resolves

Reply:
updated